whylog/tests/utils.py

```diff
@@ -8,3 +11,33 @@ def get_path_to_config_files(cls, prefix_path):
         rules_path = os.path.join(prefix_path, 'rules.yaml')
         log_type_path = os.path.join(prefix_path, 'log_types.yaml')
         return [parsers_path, rules_path, log_type_path]
+
+
+class TestRemovingSettings(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        #This change was caused by fails os.names() for new directories.
+        #Renaming was necessary to avoid conflicts between test config directories
+        #and orginal independent non tests config directories
+        SettingsFactorySelector.WHYLOG_DIR = '.whylog_test'
+        cls.remove_settings_dirs()
+
+    @classmethod
+    def tearDownClass(cls):
+        #Removed test config directories if test failed
+        cls.remove_settings_dirs()
+
+    @classmethod
+    def remove_one(clas, path):
```

Comment:
Please, rename it. It looks like it removes directory on "path". It looks very dangerous in following `remove_settings_dirs`